.eslintrc.js

```diff
@@ -28,6 +28,8 @@ module.exports = {
     '@typescript-eslint/no-non-null-assertion': 0,
     'default-case': 0,
     'import/prefer-default-export': 0,
-    'no-lonely-if': 0
+    'no-lonely-if': 0,
+    'no-shadow': 'off',
+    '@typescript-eslint/no-shadow': ['error'],
```

Comment:
Typescript complain about `no-shadow` so I follow [StackOverflow answer](https://stackoverflow.com/a/64745652/5923666) to solve this